android/library/src/main/java/com/liferay/mobile/screens/cache/Cache.java

```diff
@@ -108,6 +116,18 @@ private static synchronized Object doDatabaseOperation(Long groupId, Long userId
 		}
 	}
 
+	public static int pendingItemsToSync() {
+		int ddlFormItems = pendingItemsByClass(DDLFormEvent.class);
```

Comment:
I'm worried about having a lot of items to sync, this will block the UI thread, right?

Reply:
Do you suggest any solution? Run this in another thread?